UPGRADE-4.0.md

```diff
@@ -59,6 +59,7 @@
 - Removed `Dsn::getSecretKey()`.
 - Removed `Dsn::setSecretKey()`.
 - Removed `EventType::default()`.
+- Removed adding the value of the `logger` option as a tag on the event. If you rely on this behaviour, add the tag manually.
```

Comment:
As far as I can tell, we can still add logger on the event via the Event::setLogger() method, I think it's just the adding it via the client Client::setLogger() method that was changed to instead add a debug logger?

So I would suggest removing this sentence and fleshing out the above sentence "The logger option now accepts a Psr\Log\LoggerInterface instance instead of string" to explain that the logger tag can now be added only via the Event::setLogger() method.

Reply:
Seems I was slightly wrong about this, the Event::setLogger() method still exists, but it doesn't do anything?  So I filed PR #1657

This is important to fix because e.g. the Monolog handler has `$event->setLogger(sprintf('monolog.%s', $record['channel']));`